frontend/src/ui/player/index.tsx

```diff
@@ -123,15 +123,22 @@ export const InlinePlayer: React.FC<PlayerProps> = ({ className, event, ...playe
             // vertical space left (not taken up by the player). The height of
             // the players is actually best controlled by setting the width.
             "--ideal-max-width": `calc((90vh - 80px) * ${aspectRatio[0] / aspectRatio[1]})`,
-            maxWidth: "min(100%, var(--ideal-max-width))",
-            minWidth: "320px",
+            width: "100%",
+            minWidth: 320,
+
+            // We want to be able to see the full header, the video title and some metadata.
+            // So: full height minus header, minus separation line (18px), minus main
+            // padding (16px), minus breadcrumbs (roughly 42px), minus the amount of space
+            // we want to see below the video (roughly 90px).
+            maxHeight: "calc(100vh - var(--header-height) - 18px - 16px - 42px - 90px)",
```

Comment:
Actually, what do you think of `120px` instead of `90px`? That already brings a tiny sliver of the boxes into frame, making it more obvious that scrolling is possible.

Reply:
Unfortunately we either need to revert the changes to the player sizing or find another solution that also works for everyone's favourite special browser, safari.
The `maxHeight` works but somehow the `<InlinePlayer>` just isn't used as container for the paella player there (i.e. it doesn't limit the players size).
This can lead to really bad overflowing issues, see my screenshot.
<img width="1244" alt="Bildschirmfoto 2023-05-01 um 16 09 19" src="https://user-images.githubusercontent.com/94838646/235465078-acf7cd7b-9a61-4359-b5df-d4fc6185824f.png">